test/chrome_test.js

```diff
@@ -141,4 +141,55 @@ describe("StorageArea", function() {
     });
   });
 
+  /** @test {StorageArea#remove} */
+  describe("#remove", function() {
+    it("removes a single entry", function(done) {
+      area.set({ foo: "bar" }, function() {
+        area.remove("foo", function() {
+          area.get("foo", function(items) {
+            expect(items).to.deep.eql([]);
+            done();
+          });
+        });
+      });
+    });
+
+    it("silently ignores a missing entry", function(done) {
+      area.remove("nonexisting", function() {
+        done();
+      });
+    });
+
+    it("does not remove other entries", function(done) {
+      area.set({ foo: "bar", baz: "bar" }, function() {
+        area.remove("baz", function() {
+          area.get("foo", function(items) {
+            expect(items.foo).to.eql("bar");
+            done();
+          });
+        });
+      });
+    });
+
+    it("removes an array of keys", function(done) {
+      area.set({ foo: "bar", baz: "bar" }, function() {
+        area.remove(["foo", "baz"], function() {
+          area.get(["foo", "baz"], function(items) {
+            expect(items).to.deep.eql([]);
+            done();
+          });
+        });
+      });
+    });
+
+    it("treats the callback parameter as optional", function(done) {
+      area.set({ foo: "bar" }, function() {
+        area.remove(["foo"]);
+        setTimeout(function() {
```

Comment:
It was actually useless the way it was, but I added a timeout of 100ms now, to allow 100ms for IndexedDB to commit the transaction, see [line 204 below](https://github.com/Kinto/kinto-chrome/pull/16/files#diff-873469394a434c098126cb21ef4eec86R204)